VTEX - Marketplace Protocol - External Marketplace Mapper.json

```diff
@@ -405,335 +592,252 @@
+                                            "description": "Connector ID."
                                         },
                                         "name": {
                                             "type": "string",
-                                            "title": "name",
-                                            "example": "Mercado Livre Classic"
+                                            "description": "Connector name."
                                         },
                                         "website": {
                                             "type": "string",
-                                            "title": "website",
-                                            "example": "mercadolivre.com.br"
+                                            "description": "Connector website."
                                         },
                                         "logo": {
                                             "type": "string",
-                                            "title": "logo",
-                                            "example": "https://g.foolcdn.com/art/companylogos/square/meli.png"
+                                            "description": "Connector logo URL."
                                         },
                                         "description": {
                                             "type": "string",
-                                            "title": "description",
-                                            "example": "MercadoLibre operates online commerce platforms in Latin America. It operates Mercado Libre Marketplace, an automated online commerce platform that enables businesses, merchants, and individuals to list merchandise and conduct sales and purchases online."
+                                            "description": "Connector description."
                                         },
                                         "helpUrl": {
                                             "type": "string",
-                                            "title": "helpUrl",
-                                            "example": "https://help.vtex.com/pt/tracks/configurar-integracao-do-mercado-livre--2YfvI3Jxe0CGIKoWIGQEIq"
+                                            "description": "URL of connector integration documentation page."
                                         },
                                         "segment": {
                                             "type": "string",
-                                            "title": "segment",
-                                            "example": "OTHERS"
+                                            "description": "Market segment."
                                         },
                                         "availableCountries": {
+                                            "description": "List of countries where connector is available for integration.",
```

Comment:
Seria interessante usar a menção ao ISO:

Three-digit country code, in [ISO 3166 ALPHA-3](https://www.iban.com/country-codes) format.